lib/daru/plotting/dataframe.rb

```diff
@@ -19,7 +19,19 @@ module DataFrame
       #   # Simple bar chart
       #   df = Daru::DataFrame.new({a:['A', 'B', 'C', 'D', 'E'], b:[10,20,30,40,50]})
       #   df.plot type: :bar, x: :a, y: :b
-      def plot opts={}
+      def plot opts={}, &block
+        # TODO: Solve plot not shown when no block is passed
+        if opts[:categorized] ||
+          @index.valid?(opts[:x]) && self[opts[:x]].type == :category
```

Comment:
I don't know how to solve this issue. The plots are not working when I'm not passing any block.